packages/cosmic-proto/package.json

```diff
@@ -15,8 +15,8 @@
   },
   "type": "module",
   "exports": {
-    "./swingset/msgs.js": "./swingset/msgs.js",
-    "./swingset/msgs.ts": "./swingset/msgs.ts",
+    "./swingset/msgs.js": "./gen/agoric/swingset/msgs.js",
+    "./swingset/msgs.ts": "./gen/agoric/swingset/msgs.ts",
```

Comment:
This is probably okay but we still need the symlinks for systems that don’t support `exports` directives, like some bundlers. It would probably be safer in the face of maintenance to leave these alone and rely on the symlinks.